CleverPush/Source/CleverPushSQLiteManager.m

```diff
@@ -0,0 +1,245 @@
+                sqlite3_stmt *updateStatement;
+
+                if (sqlite3_prepare_v2(database, [updateSQL UTF8String], -1, &updateStatement, nil) == SQLITE_OK) {
+                    sqlite3_bind_int(updateStatement, 1, updatedCount);
+                    sqlite3_bind_text(updateStatement, 2, [updatedAt UTF8String], -1, SQLITE_STATIC);
+                    sqlite3_bind_text(updateStatement, 3, [bannerID UTF8String], -1, SQLITE_STATIC);
+                    sqlite3_bind_text(updateStatement, 4, [trackEventID UTF8String], -1, SQLITE_STATIC);
+                    sqlite3_bind_text(updateStatement, 5, [property UTF8String], -1, SQLITE_STATIC);
+                    sqlite3_bind_text(updateStatement, 6, [value UTF8String], -1, SQLITE_STATIC);
+                    sqlite3_bind_text(updateStatement, 7, [relation UTF8String], -1, SQLITE_STATIC);
+
+                    if (sqlite3_step(updateStatement) == SQLITE_DONE) {
+                        sqlite3_finalize(updateStatement);
+                        sqlite3_close(database);
+                        return YES;
+                    }
+                }
+            }
+        }
+
+        NSString *insertSQL = [NSString stringWithFormat:
+                               @"INSERT INTO %@ (bannerId, trackEventId, property, value, relation, count, createdAt, updatedAt, fromValue, toValue) "
+                               "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?);", cleverPushDatabaseTable];
+        sqlite3_stmt *insertStatement;
+
+        if (sqlite3_prepare_v2(database, [insertSQL UTF8String], -1, &insertStatement, nil) == SQLITE_OK) {
+            sqlite3_bind_text(insertStatement, 1, [bannerID UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_text(insertStatement, 2, [trackEventID UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_text(insertStatement, 3, [property UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_text(insertStatement, 4, [value UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_text(insertStatement, 5, [relation UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_int(insertStatement, 6, 1);
+            sqlite3_bind_text(insertStatement, 7, [createdAt UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_text(insertStatement, 8, [updatedAt UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_text(insertStatement, 9, [fromValue UTF8String], -1, SQLITE_STATIC);
+            sqlite3_bind_text(insertStatement, 10, [toValue UTF8String], -1, SQLITE_STATIC);
+
+            if (sqlite3_step(insertStatement) == SQLITE_DONE) {
+                sqlite3_finalize(insertStatement);
+                sqlite3_close(database);
+                return YES;
+            }
+        }
+        sqlite3_close(database);
+    }
+    return NO;
+}
+
+#pragma mark - to get all the recrods from database
+- (NSArray<CPAppBannerEventFilters *> *)getcleverPushDatabaseAllRecords {
```

Comment:
Do we need `cleverPush` in the function names? Cant we do it without cleverPush before or does it has a purpose?
And IF we need it, we should stay camelCase (getCleverPush.... not getcleverPush)

Reply:
I renamed all the classes related to the database and its functions.
There is no need to keep the clever push word before the functions.